packages/compass-import-export/src/export/export-json.spec.ts

```diff
@@ -47,12 +47,8 @@ describe('exportJSON', function () {
       connectionString: 'mongodb://localhost:27018/local',
     });
 
-    dropCollection = promisify(dataService.dropCollection.bind(dataService));
-
-    createCollection = promisify(
-      dataService.createCollection.bind(dataService)
-    );
-
+    dropCollection = dataService.dropCollection.bind(dataService);
+    createCollection = dataService.createCollection.bind(dataService);
```

Comment:
Nit: We might not need these as variables anymore, were they there only because of the `promisify`?

Reply:
Changed in https://github.com/mongodb-js/compass/pull/4211/commits/d15a8c8ddf6b0f66f637fe4feb1257e3c69be9d1